### PR TITLE
Remove side-effects from `main`

### LINKED ISSUE #40: Remove side-effects from `main`
Currently the main function is bloating up due to side effects from CLI commands. Refactor it to have a similar workflow to pool commands.

### PR DESCRIPTION
**Pull request by kettlebell**
Close #40

### CODE REVIEW

File: core/src/main.rs [hunk 1]
```diff
@@ -104,29 +103,20 @@ fn main() {
...
 
     // Start Oracle Core GET API Server
-    thread::Builder::new()
-        .name("Oracle Core GET API Thread".to_string())
-        .spawn(|| {
-            api::start_get_api(repost_receiver);
-        })
-        .ok();
+    //thread::Builder::new()
+    //    .name("Oracle Core GET API Thread".to_string())
```
Comment (kettlebell) [inline core/src/main.rs:110]: I think we can do away with the `api` module directly. What do you think?

**Comment by kettlebell** [comment]
Fixed up `main.rs`. In addition, I fixed some bugs in parsing the boostrap YAML file and sprinkled some logging statements in.

**Review by greenhat — CHANGES_REQUESTED** [review]
Looking great! Thank you! Let's keep local REST API running for now.

File: core/src/main.rs (hunk 1 above)
Comment (kettlebell) [reply core/src/main.rs]: Sure. The API instantiates instances of `OraclePool` which means it needs an existing oracle pool YAML config file. Therefore it makes sense to move it under the CLI `Command::Run` as I've done with the latest commit.

**Review by greenhat — APPROVED** [review]
Thank you!

**Pull request merged**